python/ray/serve/schema.py

```diff
@@ -222,7 +222,7 @@ class RayActorOptionsSchema(BaseModel):
         ),
         ge=0,
     )
-    object_store_memory: float = Field(
+    object_store_memory: int = Field(
         default=None,
         description=(
             "Restrict the object store memory used per replica when "
```

Comment:
Non-blocker: would be nice to add the unit to the description as well, probably why it was thought to be float in the first place 🙃

Reply:
I updated the description.

Reply:
I meant something like
```suggestion
            "Restrict the object store memory used per replica (in bytes) when "
```
So it's more clear it's an int and won't be thinking in terms of xxx.x KB or MB